share/spack/gitlab/cloud_pipelines/stacks/radiuss/spack.yaml

```diff
@@ -0,0 +1,128 @@
+# This file is currently being customized for quartz/toss_3_x86_64_ib.
+
+spack:
+  concretization: separately
+  view: false
+  config:
+    concretizer: clingo
```

Comment:
I don't think we want this here until the new concretizer is the default, but maybe @tgamblin has thoughts on the matter.  I just noticed this, and I now think this is actually the reason the pipeline generation job is installing a bunch of specs from source, and it's not related to using a compiler that doesn't exist in the image, as I speculated in another comment.

Reply:
Switched to the original concretizer though should probably simply remove the line.